relay-server/src/actors/processor.rs

```diff
@@ -555,6 +555,7 @@ struct InnerProcessor {
 
 impl EnvelopeProcessorService {
     /// Creates a multi-threaded envelope processor.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Should we create a helper struct like we did here?

https://github.com/getsentry/relay/blob/75397010d18044a17759161e74475297fe9b0493/relay-server/src/actors/project_cache.rs#L405-L407

Reply:
So we already have the `InnerProcessor`, if we wanna do something like this we should probably just expose it. I did that now, so let me know what you think.